node/bft/src/helpers/mod.rs

```diff
@@ -46,6 +46,9 @@ pub use signed_proposals::*;
 pub mod storage;
 pub use storage::*;
 
+pub mod telemetry;
```

Comment:
the entire module could be feature-gated to speed up compilation when it's disabled

Reply:
Resolved with [2ba93d8](https://github.com/ProvableHQ/snarkOS/pull/3516/commits/2ba93d833ab3a0017bca6a071d8260838eac4521)